packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -49,4 +49,30 @@ export class Debouncer {
       this.reject = reject;
     });
   }
+}
+
+export class I18nService{
+
+  static getTextWithDecorator(text, startTag, endTag) {
+    return `${startTag}${text}${endTag}`;
+  }
+
+  static getTextBold(text) {
+    return I18nService.getTextWithDecorator(text, '<b>', '</b>');
+  }
+
+  static getLabelNameWithCount(labelName, count) {
```

Comment:
Well, I just found it is not perfect yet. When dealing with decimal numbers, pluralization differs between english and french, and most likely other languages as well.

In french, plural is applied to values greater or equal to 2. In english, plural is applied to any value different from 1.

https://www.antidote.info/fr/blogue/enquetes/15-gramme-ou-15-grammes#:~:text=En%20fran%C3%A7ais%2C%20seules%20les%20quantit%C3%A9s,prend%20la%20marque%20du%20pluriel.

Do you think the `@salesforce/i18n` module could help with that?

Reply:
The `@salesforce/i18n` module doesn't appear to be very powerful. Mostly date/currency type things.
I added a bit of logic to be able to define plurality. But I found that certain languages also have different types of plural. (ex: Arabic has 5)
So I'm wondering if this won't be able to scale properly. Maybe we should use an `i18next` module like atomic... But then I don't know if that'll work well with these salesforce custom labels.
What do you think?

Reply:
Using `i18next` might help us. We'd have to try, but if we use `i18next` to get the suffix and Salesforce has a key with the right suffix, then it should just work, I guess.

Reply:
That would be ideal indeed. I'll look into it